crates/ruff_workspace/src/settings.rs

```diff
@@ -226,6 +234,7 @@ impl Default for FormatterSettings {
             exclude: FilePatternSet::default(),
             extension: ExtensionMapping::default(),
             target_version: default_options.target_version(),
+            per_file_target_version: CompiledPerFileVersionList::default(),
```

Comment:
I can't comment on line 251 -- thanks GitHub -- but we need to include the new file in the `Display` implementation (same for `LinterSettings`)